actortoken/client.go

```diff
@@ -9,8 +9,25 @@ import (
 	"github.com/clerk/clerk-sdk-go/v2"
 )
 
+//go:generate go run ../cmd/gen/main.go
+
 const path = "/actor_tokens"
 
+// Client is used to invoke the Actor Tokens API.
+type Client struct {
+	Backend clerk.Backend
+}
+
+type ClientConfig struct {
+	clerk.BackendConfig
+}
+
+func NewClient(config *ClientConfig) *Client {
+	return &Client{
+		Backend: clerk.NewBackend(&config.BackendConfig),
+	}
+}
+
```

Comment:
❓ Could we perhaps avoid or simplify the code generation with something like the following?

```go

var DefaultClient = &Client{Backend: clerk.GetBackend()}

func Create(ctx context.Context, params *CreateParams) (*clerk.ActorToken, error) {
  return DefaultClient.Create(ctx, params)
}
```

Reply:
:+1: Sure!